### PR TITLE
chore(starters): add dib-gatsby-starter

### PR DESCRIPTION
**Pull request by DynamisDevelopment**

### CODE REVIEW

**Comment by DynamisDevelopment** [comment]
Woops, my bad. I'll get around to it asap.

**Comment by blainekasten** [comment]
@DynamisDevelopment Did you delete your repository? Looks like this is no longer being referenced and I can't find it in your github. If you are unable to bring it back we'll have to revert this PR.

**Comment by DynamisDevelopment** [comment]
@blainekasten Nope, I built it for DropInBlog and they wanted it on their own repo. It's been moved to here, https://github.com/DropInBlog/gatsby-starter-dropinblog.

File: docs/starters.yml [hunk 1]
```diff
@@ -6067,6 +6067,22 @@
     - Pagination
     - SEO
     - Phone browser Support
+- url: https://dropinblog-gatsby-starter.netlify.app/
+  repo: https://github.com/DynamisDevelopment/dib-gatsby-starter
```
Comment (muescha) [inline docs/starters.yml:6071]: repo link is an 404

is it this here https://github.com/DropInBlog/gatsby-starter-dropinblog ?

Comment (DynamisDevelopment) [reply docs/starters.yml:6071]: You're right, and it was fixed in #24151.

**Pull request merged**